### DISCUSSION TITLE
Remove use on other option plugin

### DISCUSSION THREAD

**Discussion by Saviolbn** [runelite/runelite#17174] [Ideas]
eg: im doing a birdhouse run, i have the option to use the seed on other player, misclicking, one option would be to remove use on other players option on menu wentry swapper

**Comment by StephanBijzitter** [comment]
Custom Menu Swaps plugin:

```
# Hunter
use,* seed -> * birdhouse (empty)
```

This will not remove the option to use on other players, but will prioritize using it on the birdhouse

**Comment by ppyLEK** [comment]
> Custom Menu Swaps plugin:
> 
> ```
> # Hunter
> use,* seed -> * birdhouse (empty)
> ```
> 
> This will not remove the option to use on other players, but will prioritize using it on the birdhouse

How is average player supposed to know how to use custom swaps? Do we just assume that everyone is even able to read / understand that? edit: or even know that this is possible?

**Reply by StephanBijzitter** [reply]
The plugin contains/shows instructions when first installed, but I do agree that it could use some better examples. Either way, sharing is caring. The clan I'm in shares these in a decidated (discord) chat and _once you get used to the syntax_ it's quite easy to work with.
